pkg/core/app/app.go

```diff
@@ -100,21 +103,33 @@ func (a *App) ContainerIPv4Addr() string {
 
 func (a *App) SetupDocker() error {
 	var err error
-	cont, net, err := ParseDockerCmd(a.cmd)
-	if err != nil {
-		utils.LogError(a.logger, err, "failed to parse container name from given docker command", zap.String("cmd", a.cmd))
-		return err
-	}
-	if a.container == "" {
-		a.container = cont
-	} else if a.container != cont {
-		a.logger.Warn(fmt.Sprintf("given app container:(%v) is different from parsed app container:(%v)", a.container, cont))
-	}
 
-	if a.containerNetwork == "" {
-		a.containerNetwork = net
-	} else if a.containerNetwork != net {
-		a.logger.Warn(fmt.Sprintf("given docker network:(%v) is different from parsed docker network:(%v)", a.containerNetwork, net))
+	if a.kind == utils.DockerRun {
+		cont, net, err := ParseDockerCmd(a.cmd)
+
+		if err != nil {
+			utils.LogError(a.logger, err, "failed to parse container name from given docker command", zap.String("cmd", a.cmd))
+			return err
+		}
+		if a.container == "" {
+			a.container = cont
+		} else if a.container != cont {
+			a.logger.Warn(fmt.Sprintf("given app container:(%v) is different from parsed app container:(%v)", a.container, cont))
+		}
+
+		if a.containerNetwork == "" {
+			a.containerNetwork = net
+		} else if a.containerNetwork != net {
+			a.logger.Warn(fmt.Sprintf("given docker network:(%v) is different from parsed docker network:(%v)", a.containerNetwork, net))
+		}
+	} else if a.kind == utils.DockerStart {
+		running, err := a.docker.IsContainerRunning(a.container)
```

Comment:
How are you getting `a.container` value, you are not even parsing the docker start command to get the container name.

Reply:
It's not working for me in both cases. Whether the container is present locally or not.

Reply:
User can't give container name in docker start command, they have to use containerName flag for that. So we don't have to parse the command here.

Reply:
keploy cmd would something be like ```keployV2 record -c "docker start -a MongoApp" --containerName MongoApp --networkName keploy-network```

Reply:
But why can't we just parse because you have to specify the container name when starting a stopped container? We should parse it by default and also let the user set the value just like how we are doing in the case of docker run command.

Reply:
Will be working that then.